totalRP3/core/impl/utils.lua

```diff
@@ -1018,8 +1018,13 @@ end
 
 function Utils.music.stopSoundID(soundID, channel, source)
 	for index, handler in pairs(soundHandlers) do
-		if (not soundID or handler.id == soundID) and (not channel or handler.channel == channel) and (not source or handler.source == source) then
-			Utils.music.stopSound(handler.handlerID);
+		if (not handler.stopped) and (not soundID or soundID == "0" or handler.id == soundID) and (not channel or handler.channel == channel) and (not source or handler.source == source) then
+			if (handler.channel == "Music" and handler.handlerID == 0) then
+				StopMusic();

```

Comment:
Shouldn't this be `Utils.music.stopMusic()`?  `StopMusic()` would only stop musics played using a path, while `Utils.music.stopMusic()`stops musics both played with a path or an ID.

Reply:
It is correct. The check makes sure that the music we're trying to stop at this moment is a path music, and only stops that.
Also `Utils.music.stopMusic()` calls back `stopSoundID()` so it would go on an infinite loop.